sigs.yaml

```diff
@@ -90,6 +166,53 @@ sigs:
         description: Design Proposals
       - name: sig-apps-test-failures
         description: Test Failures and Triage
+    subprojects:
```

Comment:
I would suggest adding the Workloads API and pointing to the OWNERS files for the controllers. It's an opportunity to build for a sub-project with multiple OWNERS files.

Reply:
done